llvm/include/llvm/MC/CAS/MCCASObjectV1.h

```diff
@@ -12,10 +12,17 @@
 #include "llvm/BinaryFormat/MachO.h"
 #include "llvm/CAS/CASID.h"
 #include "llvm/CAS/ObjectStore.h"
+#include "llvm/DebugInfo/DWARF/DWARFCompileUnit.h"
+#include "llvm/DebugInfo/DWARF/DWARFContext.h"
+#include "llvm/DebugInfo/DWARF/DWARFDataExtractor.h"
+#include "llvm/DebugInfo/DWARF/DWARFDebugAbbrev.h"
+#include "llvm/DebugInfo/DWARF/DWARFDebugLine.h"
 #include "llvm/MC/CAS/MCCASFormatSchemaBase.h"
 #include "llvm/MC/CAS/MCCASReader.h"
 #include "llvm/MC/MCAsmLayout.h"
+#include "llvm/Support/DataExtractor.h"
 #include "llvm/Support/Endian.h"
+#include <unordered_set>
```

Comment:
nit: In general, we should almost always skip the std data structures and use the llvm ones. A SmallSet or DenseSet would be good choices here

Reply:
I got a bunch of errors about IsEqual not being defined when using DenseSet, so maybe it doesn't work for enums?